miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditFreezerController.java

```diff
@@ -48,6 +48,7 @@
   @RequestMapping("/new")
   public ModelAndView newArray(ModelMap model) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)